internal/entities/liquidity_provider/trusted_account.go

```diff
@@ -0,0 +1,30 @@
+package liquidity_provider
+
+import (
+	"context"
+	"errors"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+)
+
+var (
+	ErrTrustedAccountNotFound = errors.New("trusted account not found")
+	ErrDuplicateAddress       = errors.New("address already exists")
```

Comment:
I make this suggestion because we're in the trusted account context here, if you want this error to be generic that is also fine but then should be in the common file
```suggestion
	ErrDuplicateTrustedAccount       = errors.New("trusted account already exists")
```